main.py

```diff
@@ -30,9 +30,10 @@
 # Configuration of your Climate partner credentials. This assumes you have placed them in your environment. You may
 # also choose to just hard code them here if you prefer.
 
-client_id = os.environ['CLIMATE_API_ID']  # OAuth2 client ID
-client_secret = os.environ['CLIMATE_API_SECRET']  # OAuth2 client secret
-api_key = os.environ['CLIMATE_API_KEY']  # X-Api-Key
+CLIMATE_API_ID = os.environ['CLIMATE_API_ID']         # OAuth2 client ID
+CLIMATE_API_SECRET = os.environ['CLIMATE_API_SECRET'] # OAuth2 client secret
+CLIMATE_API_SCOPES = os.environ['CLIMATE_API_SCOPES'] # Oauth2 scope list
+CLIMATE_API_KEY = os.environ['CLIMATE_API_KEY']       # X-Api-Key header
 
```

Comment:
should we do any kind of checking to ensure that these are set

Reply:
yes, that's a good idea

Reply:
turns out this is already in place due to python dict access (exception is thrown if requested key is not in dict). if we want something more specific i'll add that in a separate PR